packages/eds-core-react/src/components/SideSheet/SideSheet.docs.mdx

```diff
@@ -24,6 +24,7 @@ A toggleable area that is anchored to the edge of the viewport with supplementar
     <li>Side sheets contain content that supplements the screen’s primary UI region.</li>
     <li>Side sheets display a wide variety of content and layouts.</li>
     <li>Side sheets can be visible or hidden based on screen size or user need, and they often become bottom sheets on mobile.</li>
+    <li>Content in side sheets can be broken into tabs.</li>
```

Comment:
Think this will a bit weird with our current `SideSheet` as you can't remove the title or close button? Whats the basis for this?